include/functions.h

```diff
@@ -104,11 +104,11 @@ void FaultDrawer_SetInputCallback(FaultDrawerCallback callback);
 void FaultDrawer_Init(void);
 void func_80084940(void);
 void func_80084968(void);
-void Load_Relocate(u32 allocatedVRamAddr, OverlayRelocationSection* overlayInfo, u32 vRamStart);
-s32 Load_LoadOverlay(u32 vRomStart, u32 vRomEnd, u32 vRamStart, u32 allocatedVRamAddr, u32 allocatedBytes);
+void Load_Relocate(void* allocatedVRamAddr, OverlayRelocationSection* overlayInfo, u32 vRamStart);
+size_t Load_LoadOverlay(u32 vRomStart, u32 vRomEnd, u32 vRamStart, void* allocatedVRamAddr, size_t allocatedBytes);
 void* Load_AllocateAndLoad(u32 vRomStart, u32 vRomEnd, u32 vRamStart);
-void Load2_Relocate(u32 allocatedVRamAddr, OverlayRelocationSection* overlayInfo, u32 vRamStart);
-s32 Load2_LoadOverlay(u32 vRomStart, u32 vRomEnd, u32 vRamStart, u32 vRamEnd, u32 allocatedVRamAddr);
+void Load2_Relocate(void* allocatedVRamAddr, OverlayRelocationSection* overlayInfo, u32 vRamStart);
+size_t Load2_LoadOverlay(u32 vRomStart, u32 vRomEnd, u32 vRamStart, u32 vRamEnd, void* allocatedVRamAddr);
 void* Overlay_AllocateAndLoad(u32 vRomStart, u32 vRomEnd, u32 vRamStart, u32 vRamEnd);
```

Comment:
uintptr_t's insteda of `u32`s, I think, at least for VRAM. I can't remember what we decided for VROM.